pkg/vsphere/extraconfig/keys.go

```diff
@@ -161,7 +161,7 @@ func calculateScopeFromKey(key string) []string {
 	return scopes
 }
 
-func calculateKeyFromField(field reflect.StructField, prefix string, depth recursion) (string, recursion) {
+func calculateKeyFromField(log logrus.FieldLogger, field reflect.StructField, prefix string, depth recursion) (string, recursion) {
```

Comment:
We call this from locations outside of the encode/decode paths so either we get no debug from this when those calls are set to debug or we pass the logger through.
One of the perks of moving to trace everywhere - option to use operation level debug insteadof/combinedwith component level debug.